ENIGMAsystem/SHELL/Platforms/General/PFwindow.cpp

```diff
@@ -46,6 +46,15 @@ void input_push() {
   enigma_user::mouse_hscrolls = enigma_user::mouse_vscrolls = 0;
 }
 
+void platform_text_input(std::string text) {
+  if (enigma_user::keyboard_lastkey == enigma_user::vk_backspace) {
+    if (!enigma_user::keyboard_string.empty()) enigma_user::keyboard_string.pop_back();
+    return;
+  }
```

Comment:
@JoshDreamland do you think `keyboard_key` or `keyboard_lastkey` is more correct here? They will both be updated at the same time prior to this being handled, so does it even matter? My testing also shows they behave the same.